java/src/jmri/jmrit/beantable/BeanTableDataModel.java

```diff
@@ -298,6 +298,7 @@ public void setValueAt(Object value, int row, int col) {
                             nbMan.updateBeanFromSystemToUser(nBean);
                         } catch (JmriException ex) {
                             //We should never get an exception here as we already check that the username is not valid
+                            log.error("Impossible exception setting user name");
```

Comment:
I think that, despite the “Impossible” assertion, these error messages should include the stack trace of the exception.

Reply:
Good point, will update.